pkg/api/v1/types.go

```diff
@@ -2103,6 +2103,40 @@ type NodeStatus struct {
 	NodeInfo NodeSystemInfo `json:"nodeInfo,omitempty"`
 	// List of container images on this node
 	Images []ContainerImage `json:"images",omitempty`
+	// Bounded-sized list of signatures of pods that should avoid this node, sorted
+	// in timestamp order from oldest to newest. Size of the slice is unspecified.
+	PreferAvoidPods []PreferAvoidPodsEntry `json:"preferAvoidPods,omitempty"`
+}
+
+// Describes prefer/avoid rules when scheduling pods to this node.
```

Comment:
Yes, the current comments on PreferAvoidPodsEntry is ambiguity. It should be more clear. What I want to say is “Describes a class of pods that should avoid this node" is duplicate with comments on PodSignature objects.